### ISSUE TITLE
License is unclear

### ISSUE THREAD

**Issue by pprince** [pprince/etlinefont-bower#4]
Should confirm what parts are under what licenses, and annotate the repo and releases clearly.

**Comment by graingert** [comment]
@kennysing ^

**Comment by graingert** [comment]
@kennysing it would also be great if you could license the Font under a Font-specific license: http://designshack.net/articles/typography/what-is-a-font-license-and-do-i-need-one/

**Comment by pprince** [comment]
Yup; the GPL is very specifically for licensing and software programs and source code.  It doesn't work well at all for graphics assets at all (or fonts).

I, too, strongly encourage @kennysing to seek out a more applicable license for this material.

**Comment by pprince** [comment]
## documenting for posterity...

It seems that kennysing is not the best point of contact, as he's no longer with ElegantThemes.com.

They list about 35 employees on their website (more than I expected!); Nick Roach is Founder & CEO.

They publish quite a bit more free and useful content (via the "Freebies" sections of their blog), besides this one iconfont, but their licensing terms are just as vague on the other projects as on this one.

I may try to get in touch with them, and encourage them to clarify their terms; if they do, then I think I'll package more of their stuff for npm, bower, etc.
